tests/unit/test_relation_monitors.py

```diff
@@ -97,3 +98,9 @@ def test_prometheus(self):
         # THEN alert rules are transferred to prometheus over relation data
         app_data = self.harness.get_relation_data(rel_id_prom, "cos-proxy")
         self.assertIn("alert_rules", app_data)
+
+        # AND status is "active"
```

Comment:
@przemeklal This test passes also without reordering the stored state assignment (as expected). This assertion is added here only for good measure.